### ISSUE TITLE
Live Query

### ISSUE THREAD

**Issue by o4k3n** [Sam0x90/CB-Threat-Hunting#1]
IMO, a Carbon Black Threat Hunting repo can’t be complete without a Live Query section. How else would we list registry key content or dump active connections, sockets, routing tables and firewall rules from multiple hosts at a time? 🔥

And you can’t answer Velociraptor, that’s cheating!

**Comment by Sam0x90** [comment]
You are absolutely right! I'll keep this open until I update the repository with a dedicated Live Query section.
